src/main/scala/subsystem/BusTopology.scala

```diff
@@ -2,29 +2,61 @@
 
 package freechips.rocketchip.subsystem
 
+import freechips.rocketchip.config.{Field, Parameters}
 import freechips.rocketchip.diplomacy._
 import freechips.rocketchip.tilelink._
+import freechips.rocketchip.util.Location
 
-trait HasHierarchicalBusTopology { this: BaseSubsystem =>
-  sbus.clockGroupNode := asyncClockGroupsNode
+// These fields control parameters of the five traditional tilelink bus wrappers
+case object SystemBusKey extends Field[SystemBusParams]
```

Comment:
so is the idea that these will be deprecated in the future?

Reply:
These particular ones I am fine with leaving in for backward compatibility for awhile, they're pretty harmless. The analogy here is: we still allow `Config` users to set `RocketTilesKey` manually, even though there are also ways of setting that `Field` based on reading other configuration formats...